tests/e2e/core-tests/specs/activate-and-setup/onboarding-tasklist.test.js

```diff
@@ -35,14 +35,17 @@ const runOnboardingFlowTest = () => {
 				await withRestApi.deleteAllShippingZones();
 			});
 
-			it('can reset shipping classes', async () => {
-				await withRestApi.deleteAllShippingClasses();
-			})
+			it.each(['general', 'products', 'tax'])(
```

Comment:
Kudos on noticing this one. However, one of the goals for this issue/PR was for the tests to complete the same tests before and after. This change replaces 1 test with 3 tests. 

The core suite now uses the onboarding tests from WC Admin, so you wouldn't see this in our E2E logs but the test log for this looks like:

```
  Store owner can go through store Onboarding
    ✓ can reset onboarding to default settings (86ms)
    ✓ can reset shipping zones to default settings (330ms)
    ✓ can reset to default settings (1364ms)
    ✓ can reset to default settings (1715ms)
    ✓ can reset to default settings (719ms)
    ✓ can reset shipping classes (239ms)
    ✓ can start and complete onboarding when visiting the site for the first time. (8739ms)
```

You can revert the changes to this file.

Reply:
Thanks @rrennick ok cool. Makes sense. I've now reverted the each() call in https://github.com/woocommerce/woocommerce/pull/30119/commits/7a835abaead84ec806b9c00c85657c319be00dd7 and synced this branch with `trunk`.